catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/helper/InternationalStringTypeHelper.java

```diff
@@ -85,13 +91,37 @@ public InternationalStringType create(String internationalizeThis) {
     public void setLocale(Locale locale) {
         this.locale = locale;
     }
+    
+    public void setNearestMatch(boolean findNearestMatch) {
+        this.findNearestMatch = findNearestMatch;        
+    }
 
     private Optional<String> getLocalizedString(List<LocalizedStringType> localizedStrings) {
         Optional<String> optionalLocalString = localizedStrings.stream()
                 .filter(localizedString -> locale.toLanguageTag()
                         .equals(localizedString.getLang()))
                 .findFirst()
                 .map(LocalizedStringType::getValue);
+        
+        //If an exact match has not been found then look at the base language e.g. if en-GB 
+        //has not been found then try to find the first one in the list of localizedStrings 
+        //that has the same language e.g. en-US. The behaviour of this will vary depending
+        //on the order that the localized strings are loaded
+        if (!optionalLocalString.isPresent() && findNearestMatch)
```

Comment:
This is probably going to produce unexpected behaviour, where en-AU falls back to en-GB in some cases and en-US in other cases. Perhaps specifying a fallback locale would be better than "I'll take anything in that language", which may cause unreadable results (e.g. pt-PT and pt-BR aren't mutually intelligible).